app/lib/frontend/templates/views/layout.mustache

```diff
@@ -37,9 +37,18 @@
   {{#include_survey}}
   <script async="async" defer="defer" src="//www.google.com/insights/consumersurveys/async_survey?site=5wdvu4vc5lhop3xbatxof6lzcm"></script>
   {{/include_survey}}
+  {{#oauth_client_id}}
+  <meta name="google-signin-client_id" content="{{oauth_client_id}}" />
+  <script src="https://apis.google.com/js/platform.js?onload=pubAuthInit" defer="defer"></script>
```

Comment:
Why not `async`?

Reply:
`async` makes the execution order unpredictable, and this way the `pubAuthInit` will be called after the callback is populated by our script, and the google auth is loaded and initialized.

Reply:
Okay, that's fine...